src/RadixDlt.NetworkGateway.PostgresIntegration/Metrics/SqlQueryMetricsHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace RadixDlt.NetworkGateway.PostgresIntegration.Metrics;
+
+public static class SqlQueryMetricsHelper
+{
+    public const string QueryNameTag = "QueryName";
```

Comment:
Could we please use something more... random? Something that makes it virtually impossible to accidentally detect tag presence while there's none. (think: some guid or other @#$@#)